packages/kbn-es-types/src/index.ts

```diff
@@ -7,12 +7,10 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
-import * as estypesWithoutBody from '@elastic/elasticsearch/lib/api/types';
-import type {
-  Field,
-  QueryDslFieldAndFormat,
-} from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import * as estypes from '@elastic/elasticsearch/lib/api/types';
+// TODO: Remove when all usages have been migrated to non-body
+import { SearchRequest as SearchRequestWithBodyKey } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import type { Field, QueryDslFieldAndFormat } from '@elastic/elasticsearch/lib/api/types';
```

Comment:
making no-body the default, and flagging the intention to remove the `WithBody` one when all other PRs are merged.